recipes/md4qt/all/test_package/conanfile.py

```diff
@@ -14,6 +15,14 @@ def layout(self):
 
     def requirements(self):
         self.requires(self.tested_reference_str)
+        if Version(self.version) >= "5.0.0":
```

Comment:
```suggestion
        if Version(self.dependencies["md4qt"].ref.version) >= "5.0.0":
```
The test you wrote is actually on the test recipe's version (which is not defined), not on md4qt's version